infra/resources/prod/function_profile_async.tf

```diff
@@ -0,0 +1,133 @@
+
+
+data "azurerm_key_vault_secret" "low_priority_mailup_username" {
+  name         = "low-priority-mailup-username"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+data "azurerm_key_vault_secret" "low_priority_mailup_secret" {
+  name         = "low-priority-mailup-secret"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+locals {
+  function_profile_async = {
+    name = "profas"
+    app_settings = {
+      NODE_ENV = "production"
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      // Mailup setup
+      MAILUP_USERNAME = data.azurerm_key_vault_secret.low_priority_mailup_username.value
+      MAILUP_SECRET   = data.azurerm_key_vault_secret.low_priority_mailup_secret.value
+    }
+  }
+}
+
+
+module "function_profile_async" {
+  source  = "pagopa/dx-azure-function-app/azurerm"
+  version = "~> 0"
+
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = local.location
+    domain          = local.domain
+    app_name        = local.function_profile_async.name
+    instance_number = "02"
```

Comment:
why 02?

Reply:
There is already a profile async function, although with a slightly different name, so I wanted to avoid accidentally creating a resource with the same name. Is it ok?

Reply:
yes